### PR TITLE
Promotion main

### PR DESCRIPTION
**Pull request by luchok22**
Issue #:
[Trello](https://trello.com/invite/b/AklPlVwg/ATTI9fdb094d2c14954c792b9e3930d9949a11CEF7C2/solid-practicum-dominos-team-2)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### before:
Снимок

#### after:
Снимок2

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [ ] Responsive layout and functionality have been tested
  - [ ] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
   return (
-    <>
-      <Routes>
-        <Route path='/' element={<HomePage />} />
-      </Routes>
-    </>
+    <div className="App">
+      <div className="cart-f">
+        <Routes>
+          {/* <Route path="/" element={pizza.map((item) => <Cart key={item.id} {...item} />)} /> */}
```
Comment (SolidPracticum) [inline src/App.js]: удали строку где код закомментирован

File: src/Components/Cart/Cart.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+
+const Cart = (props) => {
+    return (
+        <div className="cart">
+            <img src={props.img} alt="" />
```
Comment (SolidPracticum) [inline src/Components/Cart/Cart.jsx]: добавь название картинки в атрибут alt

File: src/Components/Cart/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            <div className="cart_b">
+                <p>{props.title}</p>
+                <div className="cart-sizes">
+                    {props.sizes.map((item) => <button>{item}</button>)}
+                </div>
+                <div className="cart-type">
+                    {props.type.map((item) => <button>{item}</button>)}
+                </div>
+                <div className="cart-price">
+                    <p>{props.price} uah</p>
```
Comment (SolidPracticum) [inline src/Components/Cart/Cart.jsx]: обязательно uah?

File: src/Components/PromotionM/PromotionM.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import imgRobot from './DominosMainPR.jpg'
+import imgCoke from './DominosMainPR2.jpg'
+import './MainPr.scss'
+const PromotionM = () => {
+    return (
+        <div className='main'>
+            <div className='item'>
+                <img src={imgRobot} alt="" />
```
Comment (SolidPracticum) [inline src/Components/PromotionM/PromotionM.jsx]: тут тоже нужно будет добавить название в атрибут alt

File: src/Components/PromotionM/PromotionM.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                <div className='item_child'>
+                <span>28-12-2022</span>
+                <h1>Київстар ТБ та Domino’s. Кіно смакує краще з Domino’s</h1>
+                <p>Сезон подаруночків триває! Оформлюй замовлення від 300 грн та автоматично отримуй доступ на «1 місяць користування Київстар ТБ з пакетом «Преміум HD» у подарунок». Акція діє з 28 грудня 2022 по 31 січня 2023 року включно. Код доступу автоматично друкується у кожному чеку до замовлення від 300 грн.</p>
+                <button>Деталi</button>
+                </div>
+                
+            </div>
+            <div className='item'>
+                <img src={imgCoke} alt="" />
```
Comment (SolidPracticum) [inline src/Components/PromotionM/PromotionM.jsx]: тут тоже

File: src/Components/Cart/Cart.jsx (hunk 3 above)
Comment (luchok22) [reply src/Components/Cart/Cart.jsx]: в доминс там вроде как валюта идет, его лучше убрать?

**Comment by Esenushka** [comment]
и реши конфликты

File: src/App.js [hunk 6]
```diff
@@ -8,6 +9,7 @@ function App() {
       <Routes>
         <Route path="/" element={<HomePage />} />
         <Route path="/drinks" element={<DrinksP />} />
+        <Route path="/promotions" element={<PromotionM/>}/>
```
Comment (Esenushka) [inline src/App.js]: в route дожны быть только компоненты с page

**Pull request merged**